tools/nnyeah/nnyeah/Reworker.cs

```diff
@@ -283,7 +288,7 @@ public void Rework (Stream stm)
 				ReworkType (type);
 			}
 			ChangeTargetFramework ();
-			RemoveXamarinReferences ();
+			ReplacePlatformAssemblyReference ();
```

Comment:
I think since this is no longer a post processing step it should happen before any other processing. This is because the rest of the processing will cause the Microsoft assembly to get imported. If we do this last we run the risk of having it in the import list twice. Does that happen? I don't know, but it seems like we'd best not tempt fate.

Reply:
Fixed.